### PR TITLE
clear cookies before logout

### PR DESCRIPTION
**Pull request by saneDG**
"Return" cookie keeps already authenticated user out of login form. This causes problem when unauthenticated user tries navigate login form but we keep sending user back to frontend app.

- Remove "Return" cookie before logout invocation.

### CODE REVIEW

**Review by NinaDang97 — APPROVED** [review]
Need @mehtis approval too to be sure

File: packages/frontend/src/routes.tsx [hunk 1]
```diff
@@ -33,6 +33,19 @@ const routes: RouteObject[] = [
...
   },
+  {
+    path: Routes.LOGOUT,
+    element: <div>LOGGING OUT...</div>,
+    errorElement: <RootBoundary />, // Send user here whenever error is thrown
+    loader: async () => {
+      console.log('Logging out');
+      document.cookie = 'Return=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
+      // check if cookie exists then redirect to /
+      if (document.cookie.indexOf('Return') === -1) {
```
Comment (mehtis) [inline packages/frontend/src/routes.tsx:44]: What if the cookie isn't removed? What then? Can they get stuck here? Should we offer a button here for manual redirect that activates after a few seconds?

Comment (mehtis) [reply packages/frontend/src/routes.tsx:44]: This comment doesn't necessarily require code changes, just some food for thought.

Comment (saneDG) [reply packages/frontend/src/routes.tsx]: Added comment about this in the code

**Review by mehtis — APPROVED** [review]

**Pull request merged**